plumpy/process_states.py

```diff
@@ -316,7 +317,7 @@ def interrupt(self, reason: Any) -> None:
     async def execute(self) -> State:  # type: ignore # pylint: disable=invalid-overridden-method
         try:
             result = await self._waiting_future
-        except Interruption:
+        except (Interruption, asyncio.CancelledError):
```

Comment:
```suggestion
        except Interruption:
```

so I assume you just want the change removed here then?
I can't remember exactly why I added it here anyway